src/polytope_blmos.jl

```diff
@@ -37,6 +37,60 @@ function is_simple_linear_feasible(sblmo::CubeSimpleBLMO, v)
     return true
 end
 
+#===============================================================================================================================#
+function bounded_compute_inface_extreme_point(sblmo::CubeSimpleBLMO, direction, x, lb, ub, int_vars; kwargs...)
+    d = []
+    for i in eachindex(direction)
+        push!(d, direction[i])
+    end
+    v = copy(x)
+    
+    fixed_idx = var_fixed_idx(lb, ub, int_vars)
+    for i in fixed_idx
+        idx = findfirst(x -> x==i, int_vars)
+        v[i] = lb[idx] 
+    end
+
+    for (idx, value) in pairs(v)
+        if value!=0 && !(idx in fixed_idx)
+            d[idx] > 0 ? v[idx] = 0 : v[idx] = 1 
```

Comment:
You will likely get syntax complaints from Julia. The correct version is
`v[idx] = d[idx] > 0 ? 0 : 1 `